docs-ref-conceptual/python-sdk-azure-install.md

```diff
@@ -69,4 +69,27 @@ If you want to install `azure` from source:
 
     git clone git://github.com/Azure/azure-sdk-for-python.git
     cd azure-sdk-for-python
-    python setup.py install
\ No newline at end of file
+    python setup.py install
+
+## Install an older version with pip
+You can install an older version of `azure` by specifying 'azure==3.0.0' version details.
+```bash
+pip install azure==3.0.0 
+```
+## Check SDK installation details with pip
+You can check `azure` SDK installation location, version details etc.
+```bash
+pip show azure # Show installed version, location details etc.
+pip freeze     # Output installed packages in requirements format.
+pip list  # List installed packages, including editables.
```

Comment:
Nit: align the comments by adding a few spaces before `# List`:

```bash
pip show azure # Show installed version, location details etc.
pip freeze     # Output installed packages in requirements format.
pip list       # List installed packages, including editables.
```

Reply:
I tried adding multiple spaces last week but it was not retaining in preview for some reason. It worked after copying whitespaces from 2nd line to  3rd line :-)